Tools/CISupport/ews-build/config.json

```diff
@@ -246,6 +248,12 @@
       "configuration": "release", "runTests": "true",
       "workernames": ["ews127", "ews128"]
     },
+    {
+      "name": "JSC-Tests-arm64-EWS", "shortname": "jsc", "icon": "buildAndTest",
```

Comment:
```suggestion
      "name": "JSC-Tests-arm64-EWS", "shortname": "jsc-arm64", "icon": "buildAndTest",
```

Reply:
You've recommended renaming from `JSC-Tests-arm64-EWS` to `jsc-arm64` in a subset of all of the references to `JSC-Tests-arm64-EWS`. Is this by design?

Reply:
Yes, for consistency with the short names for all the other queues which are lower case and very terse.

Reply:
Sounds good. I think I've made the relevant changes.